internal/handler/awsmapping.go

```diff
@@ -66,6 +66,12 @@ func AwsMappingHandler(ctx context.Context) error {
 			return nil
 		default:
 		}
+
+		if resp.Account.Status == "SUSPENDED" {
+			util.Logger.Debug(fmt.Sprintf("Suspended account detected with missing Capability access - %s (%s), skipping account", *resp.Account.Name, *resp.Account.Id), zap.String("jobName", AwsMappingName))
```

Comment:
Can you make it a warning instead?

Reply:
https://github.com/dfds/aad-aws-sync/pull/31/commits/f87996060998563ed5555f6b12a56037a444ede3